db/models/reviews.models.js

```diff
@@ -0,0 +1,16 @@
+const connection = require("../connection");
+
+exports.fetchReviews = (reviewID) => {
+    const valuesToAdd = [reviewID];
+    return connection.query(
+      `SELECT * FROM reviews
+      WHERE review_id = $1;
+      `, valuesToAdd)
+      .then((result) => {
+          if (result.rows.length === 0) {
+          
```

Comment:
Great protection against SQL injection with your parameterized query and nice to see you have included a psql error 👍